Index: v4l-utils-1.14.2/lib/libdvbv5/Makefile.am
===================================================================
--- v4l-utils-1.14.2/lib/libdvbv5/Makefile.am
+++ v4l-utils-1.14.2/lib/libdvbv5/Makefile.am	2018-05-14 13:00:54.550498880 +0200
@@ -118,7 +118,7 @@
 endif
 
 libdvbv5_la_CPPFLAGS = -I../.. $(ENFORCE_LIBDVBV5_STATIC) $(LIBUDEV_CFLAGS) $(PTHREAD_CFLAGS)
-libdvbv5_la_LDFLAGS = $(LIBDVBV5_VERSION) $(ENFORCE_LIBDVBV5_STATIC) $(LIBUDEV_LIBS) -lm -lrt
+libdvbv5_la_LDFLAGS = $(LIBDVBV5_VERSION) $(ENFORCE_LIBDVBV5_STATIC) $(LIBUDEV_LIBS) -lpthread -lm -lrt
 libdvbv5_la_LIBADD = $(LTLIBICONV)
 
 EXTRA_DIST = README gen_dvb_structs.pl
Index: v4l-utils-1.14.2/lib/libdvbv5/Makefile.in
===================================================================
--- v4l-utils-1.14.2/lib/libdvbv5/Makefile.in
+++ v4l-utils-1.14.2/lib/libdvbv5/Makefile.in	2018-05-14 13:00:45.112838058 +0200
@@ -611,7 +611,7 @@
 	descriptors/desc_partial_reception.c descriptors/desc_ca.c \
 	descriptors/desc_ca_identifier.c $(am__append_1)
 libdvbv5_la_CPPFLAGS = -I../.. $(ENFORCE_LIBDVBV5_STATIC) $(LIBUDEV_CFLAGS) $(PTHREAD_CFLAGS)
-libdvbv5_la_LDFLAGS = $(LIBDVBV5_VERSION) $(ENFORCE_LIBDVBV5_STATIC) $(LIBUDEV_LIBS) -lm -lrt
+libdvbv5_la_LDFLAGS = $(LIBDVBV5_VERSION) $(ENFORCE_LIBDVBV5_STATIC) $(LIBUDEV_LIBS) -lpthread -lm -lrt
 libdvbv5_la_LIBADD = $(LTLIBICONV)
 EXTRA_DIST = README gen_dvb_structs.pl
 all: all-am
